snapcraft/commands/daemon.start

```diff
@@ -424,6 +424,15 @@ if [ "$(stat -c '%u' /proc)" = 0 ]; then
             echo 1 > /proc/sys/kernel/unprivileged_userns_clone || true
         fi
     fi
+
+    # enable cephfs idmapped mounts support for old versions of ceph MDS
+    modprobe ceph || true
```

Comment:
I thought we agreed the other day that this would be enabled in the ubuntu kernel by default so we wouldnt need this change?

This doesn't seem appropriate to always be loading the ceph module on LXD start, even if ceph pools arent being used.

So I don't think I can accept this patch.

Reply:
>I thought we agreed the other day that this would be enabled in the ubuntu kernel by default so we wouldnt need this change?

No, it was just discussed. Of course it's easier and more flexible to just enable it in the user space.

>This doesn't seem appropriate to always be loading the ceph module on LXD start, even if ceph pools arent being used.

What's the problem with that? Having the ceph's kernel module loaded does not make any troubles or overhead. At the same time, ceph can be built-in to the kernel image.

Reply:
We dont want to be loading unnecessary modules unless they are going to be used. 
We've recently made changes to LXD to avoid loading modules related to VMs unless other /dev/kvm support is detected, for example.

Reply:
> @xnox are you OK with having that option enabled in the Ubuntu kernels by default? If so then I think that option would be preferable.

yes, as long is things work correctly on LXD stable snaps from stable tracks. Because as soon as I do it, I expose LXD snap users to it. And I cannot test this correctly. If you are asking me to do this, that's great. Because then I want to reject this proposal, and instead do it in the ubuntu kernel.

I can do this in mantic SRU too, and the hwe kernels that will roll to jammy.